mkdocs.yml

```diff
@@ -26,16 +26,17 @@ theme:
 
 nav:
   - Home: 'index.md'
-  - Quickstart: 
-    - Quickstart: 'quickstart/quickstart.md'
-    - Initialization: 'quickstart/init.md'
-    - Build and Setup: 'quickstart/build_setup.md'
-    - Run: 'quickstart/server.md'
-    - Configuration: 'quickstart/config.md'
-    - Customization: 'quickstart/customize.md'
-    - Troubleshooting: 'quickstart/troubleshoot.md'
-    - Cleanup: 'quickstart/cleanup.md'
-  - Deployment:
+  - Install: 'install/index.md'
+  - Preview: 'preview/index.md'
```

Comment:
Moderate: Preview as a name is not clear to me. In the past/other docs on Invenio this kind of "fast demo deployment" is in a *quickstart* or *get started* section so the community is quite used to this naming.

Refs:
- https://invenio.readthedocs.io/en/latest/
- https://invenio.readthedocs.io/en/latest/quickstart/index.html

Reply:
With added documentation on first page, maybe this is better? The content of the Preview section, doesn't get them "started" quickly as much as "done" quickly... 

Ultimately, this stems from my understanding of the existence of the `containerize` command. As I see it, it has 2 roles:
1- marketing: people want to have a sense of what InvenioRDM is and can do quickly
2- pre-deployment checks: developers want to see how the application would be deployed in production and want to verify the changes introduced by running it in production before doing so.

What covers these 2 roles? you *preview* before you buy and you *preview* before you deploy.

Reply:
ultra-minor: I see your point, I just don't fully like "preview" but is more than fine :) ignore this :)